### ISSUE TITLE
In "release build" I get "Error calling method of PBNI object"

### ISSUE THREAD

**Issue by ChrissDeGrece** [bruce-armstrong/pbnismtp#6]
Hi,

While testing I noticed that when I run the demo using the "Debug" build , the program runs, shows the debug messageboxes and sends email fine. When using the "release build" , when it tries to run the SetCharset method , I get "Error calling method of PBNI object".

I am using VStudio 2019 to build the project.
10.0.18362.0 SDK Version

Any ideas?

Thank you

**Comment by bruce-armstrong** [comment]
I just recompiled using VS 2019 and then called it using PB 2019 R3.  It worked fine for me.  I'm passing "iso-3356-3" for the charset.  What are you passing?

**Comment by ChrissDeGrece** [comment]
Thanks for the reply and taking the time to re-test.
It doesn't seem to care what lang code I try. If it's on release VC build it always fails and the debug VC build always works with the same options.Main diff with your test could be that I am testing with PB105?

**Comment by brucearmstrong2** [comment]
I just looked and realized that I'm not currently including support for 10.5, the oldest SDK I have in the project is 12.6.  In addition, I only support PowerBuilder samples for 2017 and 2019.

You took the code and added in the SDK for 10.5?

**Comment by ChrissDeGrece** [comment]
Yes, exactly. I used the 10.5 PBNI SDK hoping it would work for me and it kind of did.

Unfortunately I am not so proficient in C++ and hit the wall with the release build.

**Comment by brucearmstrong2** [comment]
I wouldn't be able to figure out what is going on without looking at your code.

**Comment by ChrissDeGrece** [comment]
You mean the C++ code I guess. Since it only works in debug build for me, I simply commented the #ifdef _DEBUG Messagebox functions. Didn't change anything else.

pbnismtp.cpp pastebin:  
https://pastebin.com/ZEhXcBcm

My VS project
https://1drv.ms/u/s!AnB56rrSC6WfoCtX1ZZOk_UN-Nao?e=qeSeZY

In PB I use the pbnismtpdemo105 example and have imported the n_cpp_smtp pbx (debug build)
